index.bs

```diff
@@ -52,6 +52,12 @@ dictionary StorageBucketOptions {
 </xmp>
 
 <h3 id="storage-bucket-open">Creating a bucket</h3>
+
+When not null, {{StorageBucketOptions/durability}} is a hint to the user agent
+specifying the desired {{StorageBucket/durability}}. The user agent MAY
+create a new {{StorageBucket}} with this [=bucket durability=]. The user agent
```

Comment:
Should this also include the default transaction behavior? [1] Idk if we want to further discuss how it should behave separately given [2].

[1] https://github.com/WICG/storage-buckets/blob/gh-pages/explainer.md#default-to-strict-durability
[2] https://github.com/WICG/storage-buckets/issues/3

Reply:
To follow in the footsteps of IDB, I don't think we should specify a default. A UA can be spec-conformant without any support for "strict" or any support for "relaxed", so it's not possible to enforce a default.